blueprints/addon/files/README.md

```diff
@@ -17,34 +17,6 @@ Usage
 [Longer description of how to use the addon in apps.]
 
 
-Contributing
```

Comment:
Should we link to Contributing? or is that an implied convention?

Reply:
> is that an implied convention?

yep, see https://help.github.com/articles/setting-guidelines-for-repository-contributors/

Reply:
Thanks for the link. That link reminded me that we do have a docs folder https://github.com/ember-cli/ember-cli/blob/e02ec00543/blueprints/app/files/package.json#L10

Perhaps we should move it there?

Reply:
hmm, looks like GitHub supports picking it up from there, but for visibility reason I think I'd prefer to keep it at top-level. `docs` for me is for second level documentation, where files like README or CONTRIBUTING can like to.